recipes/imagemagick/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
```

Comment:
Please rename this source to `test_package.c` + do `project(test_package C)` in `test_package/CMakeLists.txt`).
This will test whether imagemagick is really C compatible.

Reply:
bump